integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -657,6 +662,32 @@ public String getFileSystemName() {
     return mFsName;
   }
 
+  @Override
+  public void umount() {
+    // Release operation is async, we need to make sure
+    // all out stream is closed before umount the fuse
+    LOG.info("Waiting for all file out stream closed");
+    long startTime = System.currentTimeMillis();
+    while (!mCreateFileEntries.isEmpty()) {
```

Comment:
Can we use CommonUtils.waitFor() method?

Reply:
`CommonUtils.waitFor()` Only can wait one url, we need to wait all pending file out stream here